### DISCUSSION TITLE
Analyses with custom peak set

### DISCUSSION THREAD

**Discussion by fe4960** [GreenleafLab/ArchR#930] [Questions / Documentation]
Hello,

I successfully imported a peak set called independently from ArchR into an ArchRProject that has been assigned cell type clusters. Then I would like to  call marker peaks with getMarkerFeatures(), identify peak to gene/peak links with addCoAccessibility() and addPeak2GeneLinks(), and perform motif enrichment with peakAnnoEnrichment().

Because the peak length of my own peaks varies (from hundreds of bp to a few thousands of bp), I am wondering if I can still use the default parameters of the above functions? Are there any parameters I should set to normalize/consider the peak length difference between peaks?

Thanks a lot!

**Comment by rcorces** [comment]
We cannot provide advice for user-specific analyses that deviate from our own recommendations. There are no parameters built into ArchR that are explicitly designed to be used with variable-width peaks.
